Hmm...I think [build_rpm failure on Leap 15.2](https://github.com/SUSE/phoebe/runs/2446288091?check_suite_focus=true) may be due to the use of [`disabler`](https://mesonbuild.com/Reference-manual.html#disabler)?

```
DEBUG: Program python3 found: NO (disabled by: collector )
DEBUG: scripts/meson.build:1:0: ERROR: Module returned a value of unknown type.
DEBUG: A full log can be found at /builddir/build/BUILD/phoebe-0.1.1/build/meson-logs/meson-log.txt
DEBUG: error: Bad exit status from /var/tmp/rpm-tmp.ywxUuq (%build)
DEBUG:     Bad exit status from /var/tmp/rpm-tmp.ywxUuq (%build)
DEBUG: RPM build errors:
DEBUG: Child return code was: 1
INFO: EXCEPTION: [Error()]
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/mockbuild/trace_decorator.py", line 93, in trace
    result = func(*args, **kw)
  File "/usr/lib/python3.9/site-packages/mockbuild/util.py", line 600, in do_with_status
    raise exception.Error("Command failed: \n # %s\n%s" % (command, output), child.returncode)
```

Let me look into that.

--- a/scripts/meson.build
+++ b/scripts/meson.build
@@ -2,8 +2,11 @@ prog_python = import('python').find_installation(
   'python3',
   required : get_option('collector'),
   modules : ['cffi'],
-  disabler : true,
 )
+if not prog_python.found()
+  prog_python = disabler()
+endif
+
 collector_script = files('collect_stats.py')
 
 abi_src = custom_target(

